src/lib/components/submit/SubmitAddNodeProvider.svelte

```diff
@@ -90,6 +91,61 @@
 
 			debounceSave();
 		})();
+
+	export function checkFields(): boolean {
+		const validUrlDomain = 'https://wiki.internetcomputer.org/';
```

Comment:
Shouldn't it be called `validUrl`? The suffix `domain` would imply it only specifies a domain no?